vid2persona/gen/tgi_openllm.py

```diff
@@ -6,17 +6,19 @@ async def send_messages(
     hf_token: str, 
     parameters: dict
 ):
-    parameters.pop('top_k')
-    parameters['max_tokens'] = parameters.pop('max_new_token')
-    parameters['presence_penalty'] = parameters.pop('repetition_penalty')
+    parameters.pop('repetition_penalty')
+    parameters['max_tokens'] = parameters.pop('max_new_tokens')
+    parameters['logprobs'] = True
+    parameters['top_logprobs'] = parameters.pop('top_k')
```

Comment:
Perhaps, `parameters` could be validated using a decorator as well? `validate_tgi_parameters`? Not a blocker though!